apps/main/src/loan/hooks/usePageOnMount.ts

```diff
@@ -5,26 +5,32 @@ import { CONNECT_STAGE, ROUTE } from '@/loan/constants'
 import networks, { networksIdMapper } from '@/loan/networks'
 import useStore from '@/loan/store/useStore'
 import { ChainId, type NetworkEnum, PageProps, type UrlParams, Wallet } from '@/loan/types/loan.types'
-import { initCurveJs, initLendApi } from '@/loan/utils/utilsCurvejs'
+import { initLendApi, initStableJs } from '@/loan/utils/utilsCurvejs'
 import { getPath, parseNetworkFromUrl, parseParams } from '@/loan/utils/utilsRouter'
 import type { INetworkName } from '@curvefi/stablecoin-api/lib/interfaces'
 import type { ConnectState } from '@ui/utils'
 import { isFailure, isLoading, isSuccess } from '@ui/utils'
 import { getWalletChainId, getWalletSignerAddress, useSetChain, useWallet } from '@ui-kit/features/connect-wallet'
 import { REFRESH_INTERVAL } from '@ui-kit/lib/model'
+import { useApiStore } from '@ui-kit/shared/useApiStore'
 
 function usePageOnMount(chainIdNotRequired?: boolean) {
   const params = useParams() as UrlParams
   const { push } = useRouter()
   const { wallet, connect, disconnect, walletName, setWalletName } = useWallet()
   const [_, setChain] = useSetChain()
 
+  const curve = useApiStore((state) => state.stable)
+  const lending = useApiStore((state) => state.lending)
+  const updateStable = useApiStore((state) => state.updateStable)
+  const updateLending = useApiStore((state) => state.updateLending)
+  const setIsLoadingCurve = useApiStore((state) => state.setIsLoadingCurve)
```

Comment:
This app doesn't use curvejs? :thinking: 
```suggestion
```

Reply:
Good catch, actually have to use the (unused atm) setIsLoadingStable here